### PR TITLE
init

### PR DESCRIPTION
**Pull request by anmolsri1**

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <ScalaCodeStyleSettings>
+      <option name="MULTILINE_STRING_CLOSING_QUOTES_ON_NEW_LINE" value="true" />
```
Comment (arpitadash) [inline .idea/codeStyles/Project.xml:4]: wow cool.

File: com.DemoJavaProject.iml [hunk 2]
```diff
@@ -8,4 +8,7 @@
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
+  <component name="SonarLintModuleSettings">
+    <option name="uniqueId" value="159ec31e-0528-46ee-85b8-7f1ce279b083" />
```
Comment (anmolsri1) [inline com.DemoJavaProject.iml:12]:
```suggestion
  </component>
  <component name="SonarLintModuleSettings">
    <option name="uniqueId" value="159ec31e-0528-46ee-85b8-7f1ce279b083" />
```

Comment (arpitadash) [reply com.DemoJavaProject.iml:12]: ok

Comment (anmolsri1) [reply com.DemoJavaProject.iml:12]: asdfsafas

Comment (arpitadash) [reply com.DemoJavaProject.iml:12]: hahaha

Comment (anmolsri1) [reply com.DemoJavaProject.iml:12]: @revv fix the code

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (arpitadash) [reply .idea/codeStyles/Project.xml:4]: wow

File: src/DemoJavaProject/DemoJavaProject.java [hunk 3]
```diff
@@ -5,5 +5,6 @@
 public class DemoJavaProject {
     public static void main(String args[]) throws IOException {
         System.out.println("Demo Java Project");
+        // hello
```
Comment (arpitadash) [inline src/DemoJavaProject/DemoJavaProject.java:8]:
```suggestion
        // hello
```

**Pull request closed without merging**